src/Bravo3/Orm/Drivers/Redis/SentinelMonitor.php

```diff
@@ -0,0 +1,144 @@
+<?php
+namespace Bravo3\Orm\Drivers\Redis;
+
+use Bravo3\Orm\Drivers\Common\Command;
+use Predis\Client;
+
+class SentinelMonitor
+{
+    /**
+     * @var Client
+     */
+    protected $client;
+
+    /**
+     * Create a new Sentinel Monitor.
+     *
+     * If more than one sentinel server parameters are provided, the first
+     * sentinel that connects will be used for retrival of information.
+     *
+     * @param mixed $params
+     */
+    public function __construct($connection_params = null)
+    {
+        foreach($connection_params as $connection) {
+            $this->client = new Client($connection);
+
+            // Usable connection is found
+            if ($this->client->isConnected()) {
+                break;
+            }
+        }
+    }
+
+    /**
+     * Return an array of SENTINEL servers in the form that can be
+     * passed to Predis Client.
+     *
+     * @param  string $master_name
+     * @return array
+     */
+    public function findSentinels($master_name = 'mymaster')
+    {
+        $sentinels = $this->client->sentinel('sentinels', $master_name);
+        return $sentinels;
+    }
+
+    /**
+     * Return an array of MASTER servers in the form that can be
+     * passed to Predis Client.
+     *
+     * @return array
+     */
+    public function findMasters()
+    {
+        $masters = $this->client->sentinel('masters');
```

Comment:
`return $this->getConnectionParams($this->client->sentinel('masters'), 'master');`